experimental/framework/tinylicious-client/src/TinyliciousClient.ts

```diff
@@ -53,13 +56,13 @@ export class TinyliciousClientInstance {
             runtimeFactory,
             true,
         );
-        return this.getRootDataObject(container);
+        return this.getFluidContainerAndServices(container);
     }
 
     public async getContainer(
         serviceContainerConfig: TinyliciousContainerConfig,
         containerSchema: ContainerSchema,
-    ): Promise<FluidContainer> {
+    ): Promise<[container: FluidContainer, containerServices: TinyliciousContainerServices]> {
```

Comment:
Probably return an object here (and `createContainer`) so it's a little easier to change this in the future if we want.
```suggestion
    ): Promise<{container: FluidContainer, containerServices: TinyliciousContainerServices}> {
```

Reply:
I converted this into a tuple since we had decided on the OdspClient that we would be returning a tuple containing the FluidContainer, which is common for all services, and the containerServices object, which would be unique for each one (https://github.com/microsoft/FluidM365Integration/pull/10#discussion_r621458318). The latter would be the one we could keep expanding on in the future to add in more capabilities